pages/dkp/konvoy/2.1/release-notes/2.1.2/index.md

```diff
@@ -0,0 +1,126 @@
+excerpt: View release-specific information for Konvoy 2.1.2
+enterprise: false
+beta: false
+---
+**D2iQ&reg; Konvoy&reg; version 2.1.2 was released on June 1, 2022.**
+
+[button color="purple" href="https://support.d2iq.com/hc/en-us/articles/4409215222932-Product-Downloads"]Download Konvoy[/button]
+
+To get started with Konvoy, [download](../../download/) and [install](../../install/) the latest version of Konvoy.
+
+<p class="message--note"><strong>NOTE: </strong>You must be a registered user and logged on to the support portal to download this product. New customers must contact their sales representative or <a href="mailto:sales@d2iq.com">sales@d2iq.com</a> before attempting to download or install Konvoy.</p>
+
+## Release summary
+
+This release provides new features and enhancements to improve the user experience, fix reported issues, integrate changes from previous releases, and maintain compatibility and support for other packages used in Konvoy.
+
+## Fixes and Improvements
+
+### Upgrade Konvoy 1.8.4 to 2.1.1 index out of range error (COPS-7183)
+
+When attempting to upgrade a Konvoy v1.8 deployment that used an existing VPC, the `prepare-to-adopt` command triggered an error. This problem has been corrected.
+
+<!-- vale Microsoft.HeadingColons = NO -->
+### kube-oidc-proxy error: certificate signed by unknown authority (COPS-7217)
+<!-- vale Microsoft.HeadingColons = YES -->
+
+A problem that prevented successful configuration of the `kube-oidc-proxy` component when using a custom domain and TLS certificate issued by Let's Encrypt was corrected.
+
+### Konvoy-image-builder 1.7.0 fails in certain situations (COPS-7207)
+
+An issue that resulted in errors like "Failed to connect to the host via ssh" when running konvoy-image-builder v1.7.0 has been corrected. The fix is available in konvoy-image-builder v1.8.0.
+
+### KIB fails to pull images in air-gapped environments  (COPS-7198)
+<!-- vale Microsoft.Avoid = NO -->
+Running konvoy-image-builder v1.7.0 in air-gapped environments would result in failures to pull docker images if the environment used registry mirrors and/or credentials. The problem is resolved in konvoy-image-builder v1.8.0.
+<!-- vale Microsoft.Avoid = YES -->
+
+### Machine stuck in provisioned state after pivot (COPS-7166)
+
+After performing a Pivot operation to migrate cluster resources in DKP 2.1.1, some machines could be stuck in the provisioning state. This issue has been resolved.
+
+### kube-prometheus-stack error during install (COPS-7163)
+
+A problem with Prometheus where the component `kube-prometheus-stack` attempted to upgrade its CRDs to a new version with invalid specifications was resolved.
+
+### HTTP(S)_proxy variables missing from CAPA pods (COPS-7158)
+
+When deploying a 2.1.1 Konvoy cluster, the local environment variables for http_proxy, https_proxy and no_proxy were not propagated to CAPI/CAPI provider pods outside the kube-system namespace, which lead to timeouts in the CAPA pod.
```

Comment:
```suggestion
When deploying a 2.1.1 Konvoy cluster, the local environment variables for http_proxy, https_proxy and no_proxy were not propagated to CAPI/CAPA provider pods outside the kube-system namespace, which lead to timeouts in the CAPA pod.
```

Reply:
Not sure if that is supposed to be CAPI/CAPI or CAPI/CAPA

Reply:
Good catch, I committed that suggestion!